### PR TITLE
Modify pre-subinquiry to include statistical packages

### PR DESCRIPTION
**Pull request by maelle**
Fix https://github.com/ropensci/software-review-meta/issues/87#event-4989625019

The idea is to mimick the changes made by @noamross to the English template https://github.com/ropensci/software-review/commit/26735b1a91a1c7dc9f3e83776d20f6f5b8e33e34

Cc @lcolladotor @Paula-Moraga

(I'm not expecting 4 reviews but I do hope to get feedback from at least one native speaker :innocent: Thanks in advance)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Preguntar si el software está dentro del alcance de rOpenSci
-about: ¿Quieres confirmar que tu paquete encaja para revisión? Usa esta plantilla para obtener repuesta de las editoras en Español (Experimental)
+about: ¿Quieres confirmar que tu paquete encaja para revisión, o quieres envia un paquete de estadística? Usa esta plantilla para obtener repuesta de las editoras en Español (Experimental)
```
Comment (maurolepore) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:3]: `^1:` The "r" seems missing. I suggest:
"o quieres envia`{+r+}` un paquete de estadistica?"

`^2:` Also I mildly prefer:
"paquete `[-de estadistica-]{+estadistico+}`" 
I think it's like the difference between saying "package of statistics" versus "statistical package".

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 2]
```diff
@@ -22,8 +22,11 @@ Tipo de Envio: <!--submission-type-->pre-envio<!--end-submission-type-->
 
 ## Alcance 
 
-- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories) y [categorías de paquetes de estadística (Inglés)](https://stats-devguide.ropensci.org/overview.html#overview-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
```
Comment (maurolepore) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:25]: `^3:` Maybe wrap at 80 characters to make the diffs easier to see?

`^4:` Similar to `^2` I suggest (but not so sure):
"paquetes `[-de estadistica-]{+estadisticos+}`"

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 3]
```diff
@@ -36,9 +39,21 @@ Tipo de Envio: <!--submission-type-->pre-envio<!--end-submission-type-->
 	- [ ] ligamientos con software de base de datos (database software bindings)
 	- [ ] datos geoespaciales (geospatial data)
 	- [ ] análisis de texto (text analysis)
+	
+     **Packages de Estadística**
+
+	- [ ] Rutinas Bayesianas y Monte Carlo (Bayesian and Monte Carlo Routines)
+	- [ ] Reducción de dimensionalidad, Análisis de Conglomerados y Aprendizaje no supervisado (Dimensionality Reduction, Clustering, and Unsupervised Learning)
```
Comment (maurolepore) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:46]: See `^3`.

Comment (maurolepore) [reply .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:46]: `^5:` The letter case is inconsistent. Maybe stick to either "Sentence case" or "Title Case".

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 4]
```diff
@@ -36,9 +39,21 @@ Tipo de Envio: <!--submission-type-->pre-envio<!--end-submission-type-->
...
+	- [ ] Aprendizaje automático (Machine Learning)
+	- [ ] Regresión y Aprendizaje supervisado (Regression and Supervised Learning)
+	- [ ] Análisis Exploratorio de Datos (Exploratory Data Analysis (EDA) and Summary Statistics)
+	- [ ] Análisis espacial (Spatial Analyses)
+	- [ ] Análisis de series temporales (Time Series Analyses)
+
 
 - Explica cómo y por qué el paquete encaja dentro de estas categorías (1 a 3 oraciones):
 
+- Si esta enviando un paquete de estadística, [incorporó documentación de los estandares en su codigo utilizando el paquete **srr**](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr)?
```
Comment (maurolepore) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:56]: See `^2 & ^3`.

**Review by maurolepore — APPROVED** [review]
Thanks! I hit "approve" so you're ready to go once you consider my comments -- with no further review.

**Comment by maelle** [comment]
Thanks a ton @maurolepore for your thorough review!

**Pull request merged**